portal/code/cx-portal/src/types/userAdministration/UserAdministrationTypes.ts

```diff
@@ -5,3 +5,19 @@ export type InviteData = {
   email: string
   organizationName: string
 }
+
+export type TenantUser = {
```

Comment:
In the layouts of the table there are columns for "role" and "status". Can we add values for those?

Reply:
done - added both as optional values because currently they are not returned from the backend API